src/options.h

```diff
@@ -117,7 +117,7 @@
 
 // controls for when threads sleep
 #define THREAD_SLEEP_THRESHOLD_NAME     "JULIA_THREAD_SLEEP_THRESHOLD"
-#define DEFAULT_THREAD_SLEEP_THRESHOLD  1e9    // cycles (1e9==1sec@1GHz)
+#define DEFAULT_THREAD_SLEEP_THRESHOLD  4*1000*1000 // nanoseconds (4ms)
```

Comment:
```suggestion
#define DEFAULT_THREAD_SLEEP_THRESHOLD  4*1000*1000 // 4ms in nanoseconds
```
Is this what this comment is conveying?